extensions/pom.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-config-parent-1.3</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    
+    <groupId>io.smallrye.ext</groupId>
+    <artifactId>smallrye-config-extensions-1.3</artifactId>
+    
+    <packaging>pom</packaging>
+    <name>SmallRye: MicroProfile Config Extensions</name>
+    
+    <modules>
+        <module>utils/providers</module>
+        <module>utils/events</module>
+        
+        <module>sources/base</module>
+        <module>sources/abstract-file</module>
+        <module>sources/properties</module>
+        <module>sources/yaml</module>
+        <module>sources/json</module>
+        <module>sources/xml</module>
+        <module>sources/memory</module>
+        <module>sources/etcd</module>
+        <module>sources/db</module>
+        <module>sources/consul</module>
+        <module>sources/typesafe</module>
+        <module>sources/zookeeper</module>
+        
+        <module>converters/json</module>
+        
+        <module>example</module>
+        
+    </modules>
+    
+    <properties>
+        <microProfile.version>2.2</microProfile.version>
+        
+        <!-- Test -->
+        <mockito.version>2.23.4</mockito.version>
+        <junit.jupiter.version>5.3.2</junit.jupiter.version>
+        <junit.platform.version>1.3.2</junit.platform.version>
+        
+        <!-- Plugins -->
+        <surefire.version>2.22.2</surefire.version>
+        <jacoco.version>0.8.4</jacoco.version>    
+    </properties>
+    
+    <dependencyManagement>
+        <dependencies>
+            <!-- MicroProfile bom -->
```

Comment:
Am I looking at an old diff? I thought `microprofile` was removed.

Reply:
I still have the microprofile bom in dependencyManagement and then use the individual apis in the child poms without the versions

Reply:
Do you want me to remove the bom and reference the versions in the individual apis ?

Reply:
For other specs we've usually just referenced the specific APIs, like CDI and JAX-RS

Reply:
Ok cool. Will do that. Is it not difficult to keep the version in sync with the umbrella then ?

Reply:
We have them defined in the parent, so everyone can align

Reply:
Ok cool. So I should just be able to remove the bom... let me see. Will do that now.

Reply:
Done